tests/test_wps_full_rvic.py

```diff
@@ -92,19 +125,88 @@ def test_full_rvic(
     params_config_dict,
     convolve_config_dict,
 ):
-    params = (
-        f"case_id={case_id};"
-        f"grid_id={grid_id};"
-        f"run_startdate={run_startdate};"
-        f"stop_date={stop_date};"
-        f"pour_points=@xlink:href={pour_points};"
-        f"uh_box=@xlink:href={uh_box};"
-        f"routing=@xlink:href={routing};"
-        f"domain=@xlink:href={domain};"
-        f"input_forcings=@xlink:href={input_forcings};"
-        f"params_config_file=@xlink:href={params_config_file};"
-        f"convolve_config_file=@xlink:href={convolve_config_file};"
-        f"params_config_dict={params_config_dict};"
-        f"convolve_config_dict={convolve_config_dict};"
+    params = build_params(
+        case_id,
+        grid_id,
+        run_startdate,
+        stop_date,
+        pour_points,
+        uh_box,
+        routing,
+        domain,
+        input_forcings,
+        params_config_file,
+        convolve_config_file,
+        params_config_dict,
+        convolve_config_dict,
     )
     run_wps_process(FullRVIC(), params)
+
+
+@mark.slow
+@mark.online
+@mark.parametrize(
+    (
+        "case_id",
+        "grid_id",
+        "run_startdate",
+        "stop_date",
+        "pour_points",
+        "uh_box",
+        "routing",
+        "domain",
+        "input_forcings",
+        "params_config_file",
+        "convolve_config_file",
+        "params_config_dict",
+        "convolve_config_dict",
+    ),
+    [
+        (
+            "sample",
+            "COLUMBIA",
+            None,  # Date cannot be None
```

Comment:
Again, if this is the particular thing we are using to break things we should consider making that more well known.